src/client/src/rt-theme/themes.ts

```diff
@@ -116,6 +116,39 @@ export function createTheme(
       textColor: secondary[2]
     },
 
+    white: colors.spectrum.white.base,
+
+    tile: {
+      backgroundColor: primary.base,
+      textColor: secondary.base,
+      inputColor: secondary['4'],
```

Comment:
A numeric selector is fine.

Reply:
`secondary[4]`

Reply:
This is something I'm torn on. While a numeric value feels nice, it's a little less obvious to whether I'm accessing a property or an index.